build.sbt

```diff
@@ -349,7 +349,7 @@ lazy val connectClient = (project in file("spark-connect/client"))
       }
     }.taskValue,
     (Test / resourceGenerators) += Def.task {
-      val src = url("https://repository.apache.org/content/groups/public/org/apache/spark/spark-connect_2.13/4.0.0-preview1/spark-connect_2.13-4.0.0-preview1.jar")
+      val src = url("https://repository.apache.org/content/groups/snapshots/org/apache/spark/spark-connect_2.13/4.0.0-SNAPSHOT/spark-connect_2.13-4.0.0-20240830.001644-352.jar")
```

Comment:
do the owners of delta-connect know that they will be responsible for updating this over time?

the risk here is that if they do not do so, Spark 4.0 release can approach and delta-connect could be compiling against a Spark 4.0 snapshot that is months old.

Reply:
@longvu-db . Are you ok with the above? Btw this is the change you pushed to this PR to fix the connect build failures.

Reply:
@scottsand-db @vkorukanti This is something we are recently aware of and a long-term solution is being discussed.

Reply:
@vkorukanti Delta Connect should still be compilable without this `url` change.

Reply:
@longvu-db Are the snapshots going to be present all the time? Do they have any expiry after which they get deleted?

Reply:
@vkorukanti I updated the jar yesterday to be back to the 4.0 preview one but it seemed the code wasn't updated, it should be back now.

Reply:
@vkorukanti In the meantime, we are figuring a long term solution to get the latest Spark Connect codes to test Delta Connect Client.